Cabal-syntax/src/Distribution/System.hs

```diff
@@ -88,8 +88,9 @@ data ClassificationStrictness = Permissive | Compat | Strict
 --  ,HaLVM ,Hurd ,IOS, Android, Ghcjs, Wasi
```

Comment:
Commas are all over the place (e.g. “,IOS, Android”). You fixed the aliases part, fixing this too would be ace.